src/System.Management.Automation/engine/parser/Parser.cs

```diff
@@ -5010,14 +5010,20 @@ private StatementAst UsingStatementRule(Token usingToken)
                     SkipToken();
 
                     var aliasToken = NextToken();
-                    if (aliasToken.Kind == TokenKind.EndOfInput)
+                    if (aliasToken.Kind is TokenKind.EndOfInput or TokenKind.NewLine or TokenKind.Semi)
```

Comment:
`or`?

Reply:
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/patterns#logical-patterns it should work the same as the longer version: `aliasToken.Kind == TokenKind.EndOfInput || aliasToken.Kind == TokenKind.NewLine || aliasToken.Kind == TokenKind.Semi`  
If you are asking why I'm checking for EndOfInput + the others then it's because without them the parser throws an error in certain conditions.